rgw/v2/tests/aws/reusable.py

```diff
@@ -128,3 +225,75 @@ def update_aws_file_with_sts_user(sts_user_info):
     with open(root_path, "a") as file:
         parser.write(file)
     utils.exec_shell_cmd(f"cat {root_path}")
+
+
+def install_aws(root_path, ssh_con=None):
```

Comment:
We already have install_aws for ss right can we leverage that only to work for both ss and ms?

Reply:
@anrao19 , we can not use the same method, since in single site environment with our workflow we receive ssh_conn as none, with we can not use exec_command method..

where as in Multisite environment in order to fetch details from remote site, we need proceed with ssh_conn.

hence we need two approaches, currently I have combined the two approaches   in a single method: install_aws in auth.py

log with latest changes:
http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/BZ1967576/latest-test-aws.log
http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/BZ1967576/latest-version-null-delete.log